### DISCUSSION TITLE
DataspaceConnector: Failed to log message to clearing house/Failed to send contract agreement to clearing house ( String could not be parsed, could not find a boundary!)

### DISCUSSION THREAD

**Discussion by vsiop** [International-Data-Spaces-Association/DataspaceConnector#438] [Q&A] [answered]
I have deployed 2 Dataspace connectors(v8.0.1). One acting as a provider and the other as a consumer. Moreover, I have deployed the trusted connector(v6.3.0) and the clearing house service(v0.8.0). The Dataspace connectors create an agreement and then try to send the info to the clearing house. They send the info, and the clearing house receives the request but the connectors seem to receive an invalid response. 
The logs from Dataspace connector-provider:
[DataspaceConnector.docx](https://github.com/International-Data-Spaces-Association/DataspaceConnector/files/9506573/DataspaceConnector.docx)
The logs from Trusted connector:
[Trusted connector.docx](https://github.com/International-Data-Spaces-Association/DataspaceConnector/files/9506680/Trusted.connector.docx)
The logs from Clearing House:
[Clearing House.docx](https://github.com/International-Data-Spaces-Association/DataspaceConnector/files/9506698/Clearing.House.docx)

For each request from the Dataspace connector to the Clearing House the response is :
"Received invalid ids message. [exception=(String could not be parsed, could not find a boundary!)]
ids.messaging.protocol.multipart.parser.MultipartParseException: String could not be parsed, could not find a boundary"

Could you please help me why it fails? I think that the Clearing house accepts the requests but something is going wrong with the response to the Dataspace connectors.

**Comment by boxer-code** [comment, marked as answer]
Hi vsiop, did you check that all of the components are using the same daps server? Otherwise the dat send from another server is most likely not valid for another one.

**Reply by vsiop** [reply]
The problem was that Trusted connector was not able to receive a DAT
